docs/source/info_gain.rst

```diff
@@ -0,0 +1,227 @@
+        """A potential energy landscape with 4 minima."""
+        sigma = 0.12  # Width of the Gaussian wells
+        gauss1 = np.exp(-(x**2 + y**2) / (2 * sigma**2))
+        gauss2 = np.exp(-((x - 1) ** 2 + y**2) / (2 * sigma**2))
+        gauss3 = np.exp(-(x**2 + (y - 1) ** 2) / (2 * sigma**2))
+        gauss4 = np.exp(-((x - 1) ** 2 + (y - 1) ** 2) / (2 * sigma**2))
+        return -np.log(gauss1 + gauss2 + gauss3 + gauss4 + 1e-6)
+
+
+To compute the force acting on the particle, we need to compute the potential energy gradient. 
+
+.. code-block:: python
+
+    from typing import Callable
+
+    def numerical_gradient(
+        f: Callable[[float, float], float], x: float, y: float, h: float = 1e-5
+    ) -> tuple[float, float]:
+        """Compute numerical gradient using finite differences."""
+        grad_x = (f(x + h, y) - f(x - h, y)) / (2 * h)
+        grad_y = (f(x, y + h) - f(x, y - h)) / (2 * h)
+        return -grad_x, -grad_y
+
+
+This function simulates, for bopth energy landscapes, the dynamics of 100 particles for 10000 timesteps. Particles are initialized close to the minima. 
+
+.. code-block:: python
+
+    import matplotlib.pyplot as plt
+
+    def create_trajectory(
+        energy_landscape: Callable[[float, float], float], file_name: str
+    ) -> np.ndarray:
+        """Simulate Langevin Dynamics on a given energy landscape."""
+        rng = np.random.default_rng(0)
+        n_atoms = 100
+        time_steps = 10000
+        dt = 0.01  # Time step
+        diffusion_coeff = 0.6  # Diffusion coefficient (random noise strength)
+
+        if energy_landscape == energy_landscape_1:
+            particles = rng.standard_normal((n_atoms, 2)) * 0.2
+            particles[n_atoms // 2 :, 1] += 1.0
+        else:
+            particles = rng.standard_normal((n_atoms, 2)) * 0.2
+            n_group = n_atoms // 4
+            particles[n_group : 2 * n_group, 1] += 1  # (0, 1)
+            particles[2 * n_group : 3 * n_group, 0] += 1  # (1, 0)
+            particles[3 * n_group :, 0] += 1  # (1, 1)
+            particles[3 * n_group :, 1] += 1
+
+        trajectory = np.zeros((time_steps, n_atoms, 2))
+        for t in range(time_steps):
+            for i in range(n_atoms):
+                x, y = particles[i]
+                fx, fy = numerical_gradient(energy_landscape, x, y)
+                noise_x = np.sqrt(2 * diffusion_coeff * dt) * rng.standard_normal()
+                noise_y = np.sqrt(2 * diffusion_coeff * dt) * rng.standard_normal()
+
+                # Update position with deterministic force and stochastic term
+                particles[i, 0] += fx * dt + noise_x
+                particles[i, 1] += fy * dt + noise_y
+
+                trajectory[t, i] = particles[i]
+
+        plt.figure()
+        plt.plot(trajectory[:, :, 0], trajectory[:, :, 1])
+        plt.show()
+
+        dataset = np.transpose(trajectory, (1, 0, 2))
+        np.save(f"info_gain/{file_name}.npy", dataset)
```

Comment:
Try and make sure any saving of files are general to however a user wants to test the script. Setting a clear output directory that is not relative to the current working dir?